OpenRA.Game/Settings.cs

```diff
@@ -404,11 +404,11 @@ static string SanitizedName(string dirty)
 			return clean;
 		}
 
-		public static string SanitizedServerName(string dirty)
+		public string SanitizedServerName(string dirty)
 		{
 			var clean = SanitizedName(dirty);
 			if (string.IsNullOrWhiteSpace(clean))
-				return new ServerSettings().Name;
+				return $"{Player.Name}'s Game";
```

Comment:
This will need to be passed through `SanitizedName` too!

Reply:
Sanitzed it as well.